tests/ethpm/validation/test_manifest.py

```diff
@@ -78,17 +78,17 @@ def test_validate_deployments_without_deployment(manifest_with_no_deployments):
 @pytest.mark.parametrize(
     "data,expected",
     (
-        ({}, set()),
-        ([{"some": {"contractType": "one"}}], set(["one"])),
+        ([], {}),
+        ([{"some": {"contractType": "one"}}], {"one"}),
         (
             [{"some": {"contractType": "one"}, "other": {"contractType": "two"}}],
-            set(["one", "two"]),
+            {"one", "two"},
         ),
     ),
 )
 def test_extract_contract_types_from_deployments(data, expected):
     actual = extract_contract_types_from_deployments(data)
-    assert actual == expected
+    assert actual == set(expected)
```

Comment:
Alternatively, I _think_ that we can keep `actual == expected` and avoid calling `set()` on the expected sets by leaving `set()` instead of `{}` in line `81`:

```py
@pytest.mark.parametrize(
    "data,expected",
    (
        ([], set()),
        ([{"some": {"contractType": "one"}}], {"one"}),
        (
            [{"some": {"contractType": "one"}, "other": {"contractType": "two"}}],
            {"one", "two"},
        ),
    ),
)
def test_extract_contract_types_from_deployments(data, expected):
    actual = extract_contract_types_from_deployments(data)
    assert actual == expected
```

Reply:
:+1: